apps/examples/testcase/le_tc/drivers/drivers_tc_main.c

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file drivers_tc_main.c
+
+/// @brief Main Function for Drivers TestCase Example
+
+#include <tinyara/config.h>
+#include "tc_internal.h"
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int tc_drivers_main(int argc, char *argv[])
+#endif
+{
+	if (tc_handler(TC_START, "Drivers TC") == ERROR) {
```

Comment:
1. include tc_common.h to use tc_handler and TC_START/END.
2. include sys/types.h to use ERROR

Reply:
Done